### ISSUE TITLE
`a.pathname` is undefined

### ISSUE THREAD

**Issue by 5d-jh** [b-fuze/deno-dom#147]
`pathname` field is undefined when create an `a` element. I expected empty string.

```ts
import { DOMParser } from "https://deno.land/x/deno_dom/deno-dom-wasm.ts";

const document = new DOMParser().parseFromString(
  "<!DOCTYPE html><html></html>",
  "text/html"
);

const a = document.createElement('a')
console.log(a.pathname); // on browser(chrome): '', on deno-dom: undefined
```

**Comment by b-fuze** [comment]
Yeah, that's an [`HTMLAnchorElement` specific property](https://developer.mozilla.org/en-US/docs/Web/API/HTMLAnchorElement/pathname)—a subclass of `HTMLElement`—but deno-dom has only implemented its parent class `HTMLElement`. When I get some time I'll go and implement the various subclasses including `HTMLAnchorElement`

Ideally I'd like to maybe automatically generate them from Web IDL... Gotta look into that

**Comment by chaosharmonic** [comment]
This is kind of out of my depth, but it turns out [JSDOM has something for that](https://github.com/jsdom/webidl2js).
